### ISSUE TITLE
Travis-CI PR validation intermittently times out with "No output has been received in the last 10m0s"

### ISSUE THREAD

**Issue by SethTisue** [scala/scala-dev#653]
one recent example: https://gist.github.com/SethTisue/4281d9118254ac6efcfdfdb8df3454a7

another: https://github.com/scala/scala/pull/8405#issuecomment-529807902

since hitting "Restart" on Travis-CI makes the old log inaccessible (as far as I know, anyway), so a little extra effort is needed to capture these logs

**Comment by lrytz** [comment]
I can confirm seeing this very frequently.

**Comment by Jasper-M** [comment]
Another one: https://travis-ci.org/scala/scala/jobs/583094666

**Comment by ashawley** [comment]
> I can confirm seeing this very frequently.

Is it happening for 2.12 branch, as well?

**Comment by SethTisue** [comment]
another report (on 2.13) at https://github.com/scala/scala/pull/8409#issuecomment-530477065 /cc @eed3si9n

**Comment by martijnhoekstra** [comment]
I've been trying out github actions, and I haven't seen timeouts there yet.

It needs a bit of tweaking still to properly select a JDK version instead of relying on the OS provided onde, but other than that, here is an example run: https://github.com/martijnhoekstra/scala/commit/50cd647e7cd3bc08e73bbee2f0172e4e72cf08fc/checks

It's not implausible that that would be less work than finding out if/how Travis can be made to pass reliably.

**Comment by eed3si9n** [comment]
There are two kinds of job timeouts with Travis CI https://docs.travis-ci.com/user/customizing-the-build#build-timeout
1. When a job produces no log output for 10 minutes.
2. When a job on a public repository takes longer than 50 minutes.

I've seen both, but the one on  scala/scala#8409 is about the no log output variety.

**Comment by eed3si9n** [comment]
This might be due to `-warn` specified in .travis.yml  https://github.com/scala/scala/blob/64a5a5d9836e255fcb85d50a4f07835409d85b29/.travis.yml#L53

```yaml
          - sbt -Dstarr.version=$STARR -warn setupValidateTest test:compile info testAll
```

Maybe scala/scala used to print out warnings often enough (more than once every 10 minutes), but as the code gets cleaned up, there might be pockets at which there are no warnings? We could either add more warnings or make the logging more verbose by including (default) info level.

**Comment by ashawley** [comment]
Here's a 2.12 timeout from last week:

https://travis-ci.org/scala/scala/jobs/580430087

**Comment by SethTisue** [comment]
```
[info] + Parallel collections.mutable.ParHashMap[Int, Int (ectasks)].both must find or not find an element: OK, passed 100 tests.
2754[info] + Parallel collections.mutable.ParHashMap[Int, Int (ectasks)].mappings must be equal: OK, passed 100 tests.
2755[info] + Parallel collections.mutable.ParHashMap[Int, Int (ectasks)].collects must be equal: OK, passed 100 tests.
2756[info] + Parallel collections.mutable.ParHashMap[Int, Int (ectasks)].flatMaps must be equal: OK, passed 100 tests.
2757[info] + Parallel collections.mutable.ParHashMap[Int, Int (ectasks)].filters must be equal: OK, passed 100 tests.
2758
2759
2760No output has been received in the last 10m0s, this potentially indicates a stalled build or something wrong with the build itself.
2761Check the details on how to adjust your build configuration on: https://docs.travis-ci.com/user/common-build-problems/#build-times-out-because-no-output-was-received
2762
2763The build has been terminated
```

seen on 2.12 at https://travis-ci.org/scala/scala/jobs/584816882

**Comment by retronym** [comment]
From: https://github.com/scala/scala/pull/8431#pullrequestreview-290957911

```
LookupSwitch(_, _, _, _), NewArray(_, _, _), Op(_), TableSwitch(_, _, _, _, _), TypeOp(_, _), VarOp(_, _)

[warn]     val assignedInConstr = getMethod(k, "<init>").instructions.filter(_.opcode == Opcodes.PUTFIELD) map {

[warn]                                                                                                         ^

No output has been received in the last 10m0s, this potentially indicates a stalled build or something wrong with the build itself.

Check the details on how to adjust your build configuration on: https://docs.travis-ci.com/user/common-build-problems/#build-times-out-because-no-output-was-received

The build has been terminated


```

**Comment by retronym** [comment]
I'm trying the timeout increase in https://github.com/scala/scala/pull/8434

**Comment by ashawley** [comment]
It's happening for both 2.12 and 2.13 branches.  The sbt versions have been on 0.13.18 and 1.2.8 since February, respectively.  The Java version has also been at AdoptOpenJDK  1.8.0_222-b10 for a long time.   Besides the Scala code itself, that rules out a lot of the executed code, IMO.

**Comment by SethTisue** [comment]
scala/scala#8434 has been merged onto both 2.12.x and 2.13.x. presumptively closing, we can reopen if there are new reports
